File: src-old/test/java/io/tripled/social/client/infrastructure/TestSocialNetworkRepository.java
package io.tripled.social.client.infrastructure;

import io.tripled.social.client.domain.SocialNetwork;
import io.tripled.social.client.domain.SocialNetworkRepository;

public class TestSocialNetworkRepository implements SocialNetworkRepository {

  private SocialNetwork socialNetwork;

  public TestSocialNetworkRepository(SocialNetwork socialNetwork) {
    this.socialNetwork = socialNetwork;
  }

  @Override
  public SocialNetwork get() {
    return socialNetwork;
  }
}

File: src/test/kotlin/io/tripled/social/client/infrastructure/TestSocialNetworkRepository.kt
package io.tripled.social.client.infrastructure

import io.tripled.social.client.domain.SocialNetwork
import io.tripled.social.client.domain.SocialNetworkRepository

class TestSocialNetworkRepository(private val socialNetwork: SocialNetwork) : SocialNetworkRepository {
    override fun get(): SocialNetwork {
        return socialNetwork
    }
}
